content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -0,0 +1,139 @@
+=================
+Calls for Tenders
+=================
+
+**Calls for Tenders** are one of the two main types of **Purchase Agreements** available by default
+in Odoo.
+
+*Calls for Tenders* are used to request offers from multiple vendors at the same time, by inviting
+those vendors to submit offers for similar goods or services all at once. This helps companies that
+use this feature to select the cheapest and fastest vendors in the market, depending on their
+needs.
+
+While calls for tenders are primaily used by organizations in the public sector who are legally
+bound to use them when making a purchase, private companies can also use them to spend efficiently,
+as well.
+
+When multiple offers are sent to a company, they can pick the one that works best for them,
+minimizing costs and increasing profitability in the most efficient manner.
```

Comment:
When multiple offers are sent to a company, the ideal/best option can be chosen - minimizing costs and increasing profitability in the most efficient manner.